tests/foreman/ui_airgun/test_host.py

```diff
@@ -223,3 +227,37 @@ def test_positive_inherit_puppet_env_from_host_group_when_action(session):
         values = session.host.read(host.name)
         assert values['host']['hostgroup'] == hostgroup.name
         assert values['host']['puppet_environment'] == env.name
+
+
+@skip_if(settings.webdriver != 'chrome')
+@tier3
+def test_positive_export(session):
+    """Create few hosts and export them via UI
+
+    :id: ffc512ad-982e-4b60-970a-41e940ebc74c
+
+    :expectedresults: csv file contains same values as on web UI
+
+    :CaseLevel: System
+    """
+    org = entities.Organization().create()
+    hosts = [entities.Host(organization=org).create() for _ in range(3)]
+    with session:
+        session.organization.select(org.name)
+        file_path = session.host.export()
+        assert os.path.isfile(file_path)
+        with open(file_path, newline='') as csvfile:
```

Comment:
I feel like we have three different places that can possibly have different host count:
from API stand point, from UI and from file. Can we make sure that if we create 3 hosts through API, we have 3 on UI(optional) and 3 in a file(not optional)?